docs/runtimes/micronaut.md

```diff
@@ -0,0 +1,152 @@
+   ```
+2. Create a project on GitHub called `micronaut-sample-app`, and push the generated project to that repository. Change `<your-github-account>` by the name of your GitHub account:
+   ```bash
+   cd micronaut-sample-app
+   git init
+   git add .
+   git commit -m "first commit"
+   git remote add origin https://github.com/<your-github-account>/micronaut-sample-app.git
+   git branch -M main
+   git push -u origin main
+   ```
+3. In the cloned project (`cd micronaut-sample-app`), set up GitOps with NubesGen by running the NubesGen CLI ([more information here](/gitops/gitops-quick-start/)):
+   ```bash
+    ./nubesgen-cli-linux gitops
+    ```
+4. Use the command-line with NubesGen ([more information here](/reference/rest-api/)) to generate a NubesGen configuration:
+   ```bash
+   curl "https://nubesgen.com/demo.tgz?runtime=micronaut_gradle&application=app_service.standard&gitops=true" | tar -xzvf -
+   ```
+5. Push the generated terraform and GitHub action to the `main` branch of the repository:
+   ```bash
+   cd micronaut-sample-app
+   git add .
+   git commit -m "Configure GitOps with NubesGen"
+   git push -u origin main
+   ```
+6. Create a new branch called `env-dev`, and push it to trigger a build:
+   ```bash
+   git checkout -b env-dev
+   git push --set-upstream origin env-dev
+   ```
+7. Go to your GitHub project, and check that the GitHub Action is running.
+8. You can go to the [Azure Portal](https://aka.ms/nubesgen-portal) to check the created resources.
+9. The application should be deployed on your App Service instance. Its URL should be in the form `https://app-demo-XXXX-XXXX-XXXX-XXXX-dev-001.azurewebsites.net/`, and you can also find it in the GitHub Action workflow (Job: "display-information", step "Display Azure infrastructure information"), or in the Azure portal.
+As it is an empty application, you should get a 404 page with JSON describing the request.
+10. Once you have finished, you should clean up your resources:
+    1. Delete the resource group that was created by NubesGen to host your resources, which is named `rg-demo-XXXX-XXXX-XXXX-XXXX-001`.
+    2. Delete the storage account used to store your Terraform state, in the `rg-terraform-001` resource group.
+
+## Tutorial 2: running a Micronaut Guide project with a PostgreSQL database
+
+For this tutorial, we will take the Maven/Java Micronaut GraphQL guide project and deploy it to Azure using NubesGen GitOps.
+
+1. Go to the Maven/Java version of the [Micronaut Todos GraphQL guide](https://guides.micronaut.io/latest/micronaut-graphql-todo-maven-java.html).
+2. Download the [solution](https://guides.micronaut.io/latest/micronaut-graphql-todo-maven-java.html#solution) and extract it.
+3. Create a project on Github called `micronaut-graphql-todo-maven-java` and push the downloaded project to that repository. Change `<your-github-account>` by the name of your GitHub account:
```

Comment:
I would say `micronaut-graqhl-todo` instead of `micronaut-graphql-todo-maven-java`